core/iwasm/interpreter/wasm_loader.c

```diff
@@ -3,6 +3,8 @@
  * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
  */
 
+#include <stdlib.h>
```

Comment:
had better remove this line, should be unnecessary

Reply:
Our internal "include-what-you-use" tools will complain if it's not here (required by qsort.h), it should be a nop if it's already included elsewhere, right?

Reply:
Got it, we normally include all the system header files in platform layer, e.g. platform_internal.h, and don't include them in vmcore files. I think here we can change `#include "bh_common.h", #includde "bh_log.h"` below to `#include 'bh_platform.h"`. bh_platform will include all the platform header files. Could you help change it?

So does aot_loader.c.